pkg/cloudencrypt/log_test.go

```diff
@@ -0,0 +1,50 @@
+package cloudencrypt
+
+import (
+	"bytes"
+	"context"
+	"log"
+	"testing"
+
+	"github.com/keboola/go-utils/pkg/wildcards"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_LogEncryptor(t *testing.T) {
```

Comment:
```suggestion
func TestLogEncryptor(t *testing.T) {
```
It seems like it tests the encryption of logs instead of logging the encrpytion/decryption process (naming)